common/persistence/src/main/java/app/coronawarn/server/common/persistence/utils/hash/HashUtils.java

```diff
@@ -0,0 +1,92 @@
+package app.coronawarn.server.common.persistence.utils.hash;
+
+import com.google.protobuf.ByteString;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import org.springframework.util.DigestUtils;
+
+public class HashUtils {
+
+  public enum MessageDigestAlgorithms {
+    SHA_256("SHA-256");
+
+    private String name;
+
+    MessageDigestAlgorithms(String name) {
+      this.name = name;
+    }
+
+    public String getName() {
+      return name;
+    }
+  }
+
+  private HashUtils() {
+  }
+
+  /**
+   * Generates a random byte array data for a specific size.
+   *
+   * @param size - byte array size
+   * @return - random key
+   */
+  public static byte[] generateRandomByteArrayData(int size) {
+    byte[] randomKeyData = new byte[size];
+    new SecureRandom().nextBytes(randomKeyData);
```

Comment:
Maybe we need to think about security here. A SecureRandom does not produce the same random number when using the same seed. Does this provide an advantage?

Reply:
The previous calls to this generation were used exactly like this. I just moved the code in this method to be re-usable. I changed the name of it though to reflect the secure random.